### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by hackacad** [BastilleBSD/bastille#145]
Bastille 0.6 can't handle IPv6 jails out of the box

```
bastille -v 
0.6.20200202
```

**Errors when creating IPv6 jails**
```
bastille create v6jail1 12.1-RELEASE ::2
Invalid: (::2).
```

Error when adding ip6.addr manually to jail.conf
```
bastille start v6jail1
Error: IP address () already in use.
```

**Suggestion**

> create.sh 
```
validate_ip() {
    ip4=${IP}
    if expr "$ip4" : '[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*$' >/dev/null; then
    ip6=${IP}
    if expr "$ip6" : 'TODO$' >/dev/null; then
```

ip6 disable only if ipv6 is not set:
```
177c177,179
< ip6 = disable;
---
> if  [ -z "$ip6" ]
>     ip6 = disable;
> fi
```

And change the for loop to add IPv6

>start.sh (1st thoughts):
```
        ipv4=$(grep 'ip4.addr' "${bastille_jailsdir}/${_jail}/jail.conf" | awk '{print $3}' | sed 's/\;//g')
        ipv6=$(grep 'ip6.addr' "${bastille_jailsdir}/${_jail}/jail.conf" | awk '{print $3}' | sed 's/\;//g')
        if [ifconfig | grep -w "$ipv4"] | [ifconfig | grep -w "$ipv6"] >/dev/null; then
            echo -e "${COLOR_RED}Error: IP address ($ipv4$ipv6) already in use.${COLOR_RESET}"
            exit 1
        fi
```

```
/usr/sbin/sysrc -f "${bastille_jail_rc_conf}" ip6=enable
```

Then there's `rdr.sh` and `stop.sh `as well (should be similar to the 2 above)...

**Comment by hackacad** [comment]
This are just first ideas on how to add that feature.
Open to discussions

**Comment by cedwards** [comment]
Initial support added in master branch. Please provide feedback and/or close this issue as resolved.

**Comment by cedwards** [comment]
This is resolved in #154 .
